### PR TITLE
Add missing context extensions, clarify launchURL language

### PR DESCRIPTION
**Pull request by brianjmiller**
Additional work towards completing #424. 

Launch URL clarification language based on discussion from 05/13 meeting.

This includes some formatting issues to make all of the tables in this section (9.6.3) consistent.

Also includes a change that indicates "masteryScore" is determined at a registration level, though this may be unclear since the state document is intended for a session scope. The same likely needs to be done for moveOn, etc.

### CODE REVIEW

**Comment by bscSCORM** [comment]
👍

**Comment by cawerkenthin** [comment]
@brianjmiller,
In 9.6.3.2 "Description: the masteryScore is for a single AU, not the entire registration.  So the language change here confuses me.

**Comment by cawerkenthin** [comment]
@brianjmiller 
In 9.6.3.3, what changed?

**Comment by brianjmiller** [comment]
9.6.3.3 is whitespace only changes, the table was shifted over a space different than the others.

File: cmi5_spec.md [hunk 1]
```diff
@@ -925,70 +928,113 @@ Used to identify statements from the AU using the publisher's id from the course
...
 </table>
 
 <a name="context_extensions_masteryScore"></a>
 ####9.6.3.2 masteryScore
 
 <table>
   <tr><th align="right" nowrap>ID:</th><td>https://w3id.org/xapi/cmi5/context/extensions/masteryscore</td></tr>
-  <tr><th align="right" nowrap>Description:</th><td>"masteryScore" as provided in the LMS Launch Data for the session used to determine the pass/fail result based on score</td></tr>
-  <tr><th align="right" nowrap>LMS Usage:</th><td></td></tr>
+  <tr><th align="right" nowrap>Description:</th><td>"masteryScore" as provided in the LMS Launch Data for the registration used to determine the pass/fail result based on score</td></tr>
```
Comment (MrBillMcDonald) [inline cmi5_spec.md]: "masteryScore" as provided in the LMS Launch Data for AU plus registration used to determine the pass/fail result based on score.

**Comment by brianjmiller** [comment]
I've added a commit to address comments from meeting on 05/20. I found a few additional issues with the section in so doing, so they've been included in the commit.

**Pull request merged**